### ISSUE TITLE
Check external pods in K8s

### ISSUE THREAD

**Issue by JoanFM** [jina-ai/jina#4145]
**Describe the feature**
After #4089 and #4130 are merged. We should make sure the user experience of connecting to a sandbox external Executor is good

**Comment by jacobowitz** [comment]
Make sure that:

- External pods are not part of the ressources created by us
- Gateway can communicate with the external pod
